packages/frontend/src/pages/VerifyEmail.tsx

```diff
@@ -74,41 +57,37 @@ export const VerifyEmailPage: FC = () => {
             <Helmet>
                 <title>Verify your email - Lightdash</title>
             </Helmet>
-            <LogoWrapper>
-                <Logo src={LightdashLogo} alt="lightdash logo" />
-            </LogoWrapper>
-            <FormWrapper>
-                <CardWrapper elevation={2}>
-                    <VerifyEmailForm
-                        email={user.data?.email || 'your e-mail.'}
-                        methods={methods}
-                        data={data}
-                        expirationTime={data?.otp?.expiresAt || new Date()}
-                        onSubmit={({ code }) => {
-                            mutate(code);
-                        }}
-                        onResend={sendVerificationEmail}
-                        isLoading={statusLoading || emailLoading}
-                        verificationLoading={verificationLoading}
-                    />
-                </CardWrapper>
-                <FormFooterCopy>
+            {/* FIXME: use Mantine sizes for width */}
+            <Stack w={400} mt="xl" pt="lg">
```

Comment:
you can rebase on main and use something from this list:
```
        one: rem(1),
        two: rem(2),
        xxs: rem(4),
        xs: rem(8),
        sm: rem(12),
        md: rem(16),
        lg: rem(20),
        xl: rem(24),
        xxl: rem(32),
        '3xl': rem(40),
        '4xl': rem(48),
        '5xl': rem(64),
        '6xl': rem(80),
        '7xl': rem(96),
        '8xl': rem(128),
        '9xl': rem(160),
```

Reply:
done 👍🏼